core/src/main/java/hudson/Functions.java

```diff
@@ -2289,4 +2289,9 @@ public static boolean isContextMenuVisible(Action a) {
             return true;
         }
     }
+
+    @Restricted(NoExternalUse.class)
+    public static String generateItemId(String id) {
+        return Math.floor(Math.random() * 3000) + "-" + id;
```

Comment:
i think you can suppress the warning unless you want to change it to something else, it's not relevant in this context